### PR TITLE
hwdef: add YJUAV_A6Ultra board support

### PR DESCRIPTION
**Pull request by yunjiuav**

### CODE REVIEW

**Comment by yunjiuav** [comment]
@Hwurzburg @andyp1per Please help me check it. thanks.

File: libraries/AP_HAL_ChibiOS/hwdef/YJUAV_A6Ultra/hwdef.dat [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+PD4 USART2_RTS USART2
+
+# USART6 TELEM2
+PC7 USART6_RX USART6
+PC6 USART6_TX USART6
+PG13 USART6_CTS USART6
+PG8 USART6_RTS USART6
+
+# USART3 GPS1
+PD9 USART3_RX USART3 NODMA
```
Comment (andyp1per) [inline libraries/AP_HAL_ChibiOS/hwdef/YJUAV_A6Ultra/hwdef.dat:42]: I'm not sure you really need to switch off DMA on both of these ports - you don't have that many serial ports. But its your call

**Review by andyp1per — APPROVED** [review]
LGTM

**Review by Hwurzburg — APPROVED** [review]

File: libraries/AP_HAL_ChibiOS/hwdef/YJUAV_A6Ultra/hwdef.dat (hunk 1 above)
Comment (yunjiuav) [reply libraries/AP_HAL_ChibiOS/hwdef/YJUAV_A6Ultra/hwdef.dat:42]: This is a good suggestion, I will submit the override changes now, thank you.

**Comment by yunjiuav** [comment]
It can be merged, thanks. The wiki will push it when I push the merge of version 4.4.5.

**Comment by peterbarker** [comment]
Merged, thanks!

**Pull request merged**